doc/api/feature-flags.md

```diff
@@ -54,10 +54,10 @@ const (
     AnnotationFeatureEnforcementMode       = AnnotationFeaturePrefix + "enforcement-mode"
 
     // CSI.
-    AnnotationFeatureMaxFailedCsiMountAttempts      = AnnotationFeaturePrefix + "max-csi-mount-attempts"
-    AnnotationFeatureMaxCsiMountTimeout             = AnnotationFeaturePrefix + "max-csi-mount-timeout"
-    AnnotationFeatureReadOnlyCsiVolume              = AnnotationFeaturePrefix + "injection-readonly-volume"
-    AnnotationFeatureCodeModulesRemoteImageDownload = AnnotationFeatureCodeModulesPrefix + "remoteImageDownload"
```

Comment:
```
➜  dynatrace-operator git:(feature/download-via-job) rg AnnotationFeatureCodeModulesRemoteImageDownload
➜  dynatrace-operator git:(feature/download-via-job)
```
return nothing so we removed/renamed old feature flag using new one? is it accurate?

Reply:
only the name of the `var` and related function was renamed, this function/ff was not yet used